### DISCUSSION TITLE
Table of contents doesn't work

### DISCUSSION THREAD

**Discussion by ghost** [adityatelange/hugo-PaperMod#232] [Q&A]
Hello, I've been trying to include a TOC in my post. I did this by speficyfing ShowToc: true. This works perfectly so a nice TOC is displayed in the top of the post: 

![image](https://user-images.githubusercontent.com/35471109/107707707-e8f9c580-6c87-11eb-9405-1d2bd04d107e.png)

But if you click in some section it doesn't direct you to that section. I'm a beginner so maybe I've not looked enough for the answer to this problem, sorry for that and thanks a lot.

**Comment by adityatelange** [comment]
Can you share us your website source/URL and link it here ? There isn't much info here, are you using something else like blog down ?

**Reply by ghost** [reply]
I'm working with blog down in R, here is an example of one of my posts: https://r-in-the-lab.netlify.app/1/01/01/two-treatments-comparison/

The previous image was a default example generated with blog down and the function serve_site.

Thank you

**Reply by adityatelange** [reply]
#206 was something related, maybe ? I cannot help coz I don't use blogdown, sorry.

**Reply by ghost** [reply]
I don't know if it's the best solution, I use the next function within my Rmd files and works well:

https://gist.github.com/gadenbuie/c83e078bf8c81b035e32c3fc0cf04ee8

Maybe it could be useful for someone with a related problem.

Thanks again for your responses, it's a very nice theme :)

**Comment by dkibalnikov** [comment]
I experienced the same problem using blogdown rendering method to html. I managed to fix problem by switching to ` blogdown.method = 'markdown'` in .Rprofile in the root directory of the project

**Reply by jeffkimbrel** [reply]
This worked for me... thanks! But, I notice any html in the markdown (such as `<span>` tags) do not get rendered. So, as long as there is only pure markdown and `{r}` chunks, this is a good fix.

**Reply by ghost** [reply]
Thanks! Recently I also found useful to put this on the YAML header on my rmarkdown document:
```
output:
  blogdown::html_page:
      toc: TRUE
```
This will display a open TOC without the option of close it as the provided by PaperMod theme
